### PR TITLE
Python3 port of tklamq

### PR DESCRIPTION
**Pull request by JedMeister**
Python3 port of tklamq for v17.x

@ongle @qq7

### CODE REVIEW

File: tklamq_lib/amqp.py [hunk 1]
```diff
@@ -156,11 +165,10 @@ def decode_message(message_data, secret=None):
     """
     sender = str(message_data['sender'])
     content = base64.urlsafe_b64decode(str(message_data['content']))
-    timestamp = datetime(*map(lambda f: int(f), message_data['timestamp-utc']))
+    timestamp = datetime(*[int(f) for f in message_data['timestamp-utc']])
```
Comment (OnGle) [inline tklamq_lib/amqp.py:168]: You can actually get even more simplified by using `map` but ditching the lambda (as the lambda was doing nothing other than calling int, so we can just use int directly) `datetime(*map(int, message_data['timestamp-utc']))`

File: tklamq-publish [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    secret = None
+    content = ''
+    if args.encrypt:
+        secret = os.getenv('TKLAMQ_SECRET', None)
+        if not secret:
+            fatal('TKLAMQ_SECRET not specified, cannot encrypt')
+
+    if args.input:
+        #if args.input == '-':
```
Comment (OnGle) [inline tklamq-publish:77]: Regression, no longer handles stdin

**Review by OnGle — COMMENTED** [review]
Looks good although I'd like clarification on why stdin usage is no longer supported.

File: tklamq_lib/amqp.py (hunk 1 above)
Comment (JedMeister) [reply tklamq_lib/amqp.py:168]: Ah ok, thanks. FWIW I think that this is another example of `2to3` giving code that works, but isn't quite ideal...

File: tklamq-publish (hunk 2 above)
Comment (JedMeister) [reply tklamq-publish:77]: Yep, it's a known regression. I did try briefly to get it working, but couldn't get it to work as I thought it should. AFAIK, that functionality isn't strictly required currently, so it shouldn't be a big deal for now. Having said that, I will open an issue.

**Comment by JedMeister** [comment]
Argh! I forgot to push my most recent changes before I merged. It's not ideal, but I just cherry picked it and merged it in on top. FYI, the commit I added is: https://github.com/turnkeylinux/tklamq/commit/ce0db6ee907e72170baec72acd0f7f560009f243

**Pull request merged**